tests/test_datayaml.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+import pytz
+import yaml
+
+from datetime import datetime
+
+
+KEYS_CONTAINING_DATES = ['last_updated', 'expires', 'sent']
+
+
+def get_alerts_data(path):
+    with open(path) as file:
+        alerts_data = yaml.load(file, Loader=yaml.FullLoader)
+    return alerts_data
+
+
+def does_date_contain_explicit_offset(date):
+    if not isinstance(date, datetime):
+        return False
+    localised_datetime = pytz.timezone('Europe/London').localize(date.replace(tzinfo=None))
```

Comment:
Minor: we could make this a bit clearer / more explicit:

```
zoneless_date = date.replace(tzinfo=None)
expected_offset = zoneless_date.astimezone(timezone('Europe/London')).utcoffset()
actual_offset = date.utcoffset()
```

Reply:
ok, will do!